logging/doc.go

```diff
@@ -146,5 +146,32 @@ parent.Timestamp marks the end of the request.)
 You should observe the child log entries grouped under the parent on the console. The
 parent entry will not inherit the severity of its children; you must update the
 parent severity yourself.
+
+# Automatic Metadata Extraction
```

Comment:
Do we want to call this "Automatic Metadata Extraction" or "Automatic Trace/Span Id Extraction" as in https://github.com/googleapis/python-logging/pull/915?

Reply:
I think it would be good to change it to be more specific in this instance.